Remove problematic part from `unbind` docs (#2570)
neo4j/docs-operations: modules/ROOT/pages/clustering/clustering-advanced/unbind.adoc

Use the `unbind` command only when troubleshooting **a specific server** and remember there is no guarantee that the allocator will reassign the same databases to this server, potentially resulting in orphaned database stores.

> The `unbind` command preserves all database stores on the server; and when the unbound server is restarted, it is seen as an entirely new server - so it won't host any of the database it did host before the operation.

Reviewer: ```suggestion
The `unbind` command preserves all database stores on the server; and when the unbound server is restarted, it is seen as an entirely new server.
Therefore, it will not host any of the databases it hosted before the operation.
```

Reviewer: Should we add here that you can manually re-attach databases to the server? Database stores are still there.

> The `unbind` command can't be used anymore to convert a cluster member into a standalone server. For that we recommend taking backups, create the standalone server and the use those backups to create the databases.

Reviewer: ```suggestion
In 2025.x, the `unbind` command cannot be used to convert a cluster member into a standalone server.
Instead, it is recommended to take backups, create the standalone server, and then use those backups to restore the databases.
```

Reviewer: For the 5.x docs, I'll use 'In 5.x series, the `unbind` command cannot be used...'
